### ISSUE TITLE
Just a thank you!

### ISSUE THREAD

**Issue by via80HD** [bravepipeproject/BravePipeExtractor#97]
Just wanted to say thank you for putting in the time and effort  to add Rumble & Bitchute to the app. I've been thinking about this idea and wishing I had this this exact feature. I wish I knew enough about programming to be able to help you out with the project but unfortunately, my brain doesn't work that good....

Anyways, thanks again for what you're doing here

**Comment by shanedroid** [comment]
Likewise thank you @evermind-zz for the work you have done here. I have no desire to authenticate and use features like comments/likes/history etc from video streaming platforms. I have been a long time NewPipe user on Android and was disappointed by the [devs stance on not allowing alternative sources like Rumble](https://github.com/search?q=teamnewpipe+rumble&type=issues) but alas it is their project and their right to choose how to invest their time on it, they built arguably one of the best FOSS mobile video streaming clients out there! Please consider adding sponsorship options to this project, I will gladly contribute to thank you for your efforts/time. Thank you!!!

**Comment by JeffersM1** [comment]
Thanks for providing a great product. I have been wanting something like this for a couple of years the could play streams from different streaming sites. Its so much easier to have all subscriptions monitored from one place.

I think it would be nice to be able to use features like comments/likes/history to show engagement with the stream content provider, but I think those features should come from the main New Pipe project.

**Comment by evermind-zz** [comment]
@all thank you for the appreciation.

> I think it would be nice to be able to use features like comments/likes/history to show engagement with the stream content provider, but I think those features should come from the main New Pipe project.

As far as I know NewPipe does not want to support any login mechanism for privacy reasons. I struggle to find much time for BraveNewPipe at all so such a feature will not be implemented by myself at the moment. If anyone want to contribute I will try to include it.

**Comment by JeffersM1** [comment]
Recently I have started downloading audio only files to save on mobile data.  This lets me listen to the files while I am doing yard work.  The audio download works for YouTube, but doesn't seem to be an option for Rumble.  Is it that Rumble doesn't support audio only downloads?  On the other hand, the Rumble audio only streaming seems to work just fine as well as the Rumble video file download.

**Comment by evermind-zz** [comment]
> Recently I have started downloading audio only files to save on mobile data. This lets me listen to the files while I am doing yard work. The audio download works for YouTube, but doesn't seem to be an option for Rumble. Is it that Rumble doesn't support audio only downloads? On the other hand, the Rumble audio only streaming seems to work just fine as well as the Rumble video file download.

Exactly, Rumble has no support for audio only. Background audio streaming is still using a video stream on Rumble just ignoring the video.

**Comment by daffydock** [comment]
I was hoping that someone would have stated this and I am happy to see that @via80HD and @shanedroid had, before I got here. 

I do a lot research for multiple subjects both for work and for fun/curiosity and I find YT to be rather restrictive at times. Even on just humour. I follow a number of fairly vanilla content creators that have gotten their videos either demonetized and/or taken down due to whatever new scope or filter YT has decided to implement that week. Some of them have moved to Rumble or post there, too. As well as other platforms.  

Satire is dying on YT and on other platforms because someone, somewhere in the world may watch a video and get overly offended, because sheer statistics say that it will happen and it does. Especially these days. 

I get YT wants to make money and due to it wants to be their version of extra-family friendly and that is fine but it comes down to the argument of whether YT and others are just service platforms or are part of the public square. Not to mention their recent desire to ban ad-blockers tells me that looking into other platforms, even on mobile is probably the better way going forward. 

Sure, the other platforms have a lot of trash in them too, but also have a lot of nuggets and leads to different topics that come with the territory of less censorship. I personally think it is worth the trade-off. It is unfortunate that NewPipe Devs decided to go in the direction they went. As it seemed to be mostly a political choice, forgetting that the rest of the world does not see politics through the lenses or prism of the USA, nor necessarily cares to do so.  Or that not everyone is a gullible idiot. 

However, it is _great_ that BraveNewPipe has some these platforms within the same app. I use it daily, it is a time saver, and I do not need to use different apps to get to the info/videos I want. It also makes bookmarking and doing backups less complicated. 

Thanks I lot @evermind-zz for taking NewPipe in this direction.  Cheers.
